src/test/java/g0301_0400/s0357_count_numbers_with_unique_digits/SolutionTest.java

```diff
@@ -0,0 +1,19 @@
+package g0301_0400.s0357_count_numbers_with_unique_digits;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class SolutionTest {
+
```

Comment:
Please remove empty line

Reply:
the empty line appear when I run 'gradlew.bat clean spotlessJavaApply test ' command

Reply:
![image](https://user-images.githubusercontent.com/93962044/145915469-a81628de-9f95-46fd-bfed-c62e7fe66fad.png)
I think we need to add the empty line to pass the check

Reply:
Please remove empty line only before @Test then format sources

Reply:
> the empty line appear when I run 'gradlew.bat clean spotlessJavaApply test ' command

Ok, let's leave empty line.